### PR TITLE
refactor: Replace Typography with Text component in skip-srp-backup-popover.js

### PR DESCRIPTION
**Pull request by Devin-Apps**
## **Description**

Migrate Typography to Text in `skip-srp-backup-popover.js`

Devin Run Link: https://preview.devin.ai/devin/c985a865ba6044c9bf43ecceea4433ac

[Open in GitHub Codespaces](https://codespaces.new/MetaMask/metamask-extension/pull/25149?quickstart=1)

## **Related issues**

Partially Fixes: https://github.com/MetaMask/metamask-extension/issues/17670

## **Manual testing steps**

1. Go to the latest build of storybook in this PR
2. Manually check if `SkipSRPBackupPopover` is rendered properly.

### **Before**
Screenshot 2024-06-13 at 10 11 31

### **After**

Screenshot 2024-06-13 at 09 42 19

## **Pre-merge author checklist**

- [X] I’ve followed [MetaMask Coding Standards](https://github.com/MetaMask/metamask-extension/blob/develop/.github/guidelines/CODING_GUIDELINES.md).
- [X] I've completed the PR template to the best of my ability
- [X] I’ve included tests if applicable
- [X] I’ve documented my code using [JSDoc](https://jsdoc.app/) format if applicable
- [X] I’ve applied the right labels on the PR (see [labeling guidelines](https://github.com/MetaMask/metamask-extension/blob/develop/.github/guidelines/LABELING_GUIDELINES.md)). Not required for external contributors.

## **Pre-merge reviewer checklist**

- [x] I've manually tested the PR (e.g. pull and build branch, run the app, test code being changed).
- [x] I confirm that this PR addresses all acceptance criteria described in the ticket it closes and includes the necessary testing evidence such as recordings and or screenshots.

### CODE REVIEW

**Comment by Devin-Apps** [comment]
Tagging @georgewrmarshall for review

File: ui/pages/onboarding-flow/secure-your-wallet/skip-srp-backup-popover.js [hunk 1]
```diff
@@ -105,12 +99,12 @@ export default function SkipSRPBackup({ handleClose }) {
               checked={checked}
               dataTestId="skip-srp-backup-popover-checkbox"
             />
-            <Typography
+            <Text
               className="skip-srp-backup-popover__details"
-              variant={TypographyVariant.H7}
+              variant={TextVariant.bodyXs}
```
Comment (georgewrmarshall) [inline ui/pages/onboarding-flow/secure-your-wallet/skip-srp-backup-popover.js:104]: I think we probably want to use norm body here which is the default for Text so we can remove
```suggestion
```

**Review by georgewrmarshall — CHANGES_REQUESTED** [review]
Looking good! Sometimes the mapping isn't always right. Here we probably want the text to be 16px of bodyMd

File: ui/pages/onboarding-flow/secure-your-wallet/skip-srp-backup-popover.js (hunk 1 above)
Comment (Devin-Apps) [reply ui/pages/onboarding-flow/secure-your-wallet/skip-srp-backup-popover.js]: Removed `variant={TextVariant.bodyXs}`

**Review by georgewrmarshall — APPROVED** [review]
LGTM! 🚀

**Review by garrettbear — APPROVED** [review]

**Pull request merged**